src/web/web.ts

```diff
@@ -28,8 +28,8 @@ app.get("/*", (req, res) => {
 
 export const start = (port: number): Promise<void> => {
   return new Promise<void>((resolve, reject) => {
-    const server = app.listen(process.env.PORT || 5000, function() {
-      console.log(`Listening on port ${process.env.PORT} 3000`);
+    const server = app.listen(process.env.PORT || 5001, function() {
+      console.log(`Listening on port ${process.env.PORT} 8000`);
```

Comment:
Why 5001 port?

Also ```console.log(`Listening on port ${process.env.PORT} 8000`);``` will always output 8000.
Better rewrite it to something like:
```
const port = process.env.PORT || 5001;
const server = app.listen(port, function() {
  console.log(`Listening on port ${port}`);
}
```

Reply:
What is process.env.PORT? Code reviewing suppose at least understanding of the code, being reviewed.

Reply:
process.env.PORT is needed for heroku deployment. Back-end will be refactored during back-end develpoment process.